x-pack/plugins/security_solution_ess/public/upselling/register_upsellings.tsx

```diff
@@ -101,8 +114,20 @@ export const upsellingPages: (services: Services) => UpsellingPages = (services)
 ];
 
 // Upsellings for sections, linked by arbitrary ids
-export const upsellingSections: UpsellingSections = [
+export const upsellingSections: (services: Services) => UpsellingSections = (services) => [
   // It is highly advisable to make use of lazy loaded components to minimize bundle size.
+  {
+    id: 'entity_analytics_panel',
+    minimumLicenseRequired: 'platinum',
+    component: () => (
+      <EntityAnalyticsUpsellingSectionLazy
+        requiredLicense="Platinum"
+        subscriptionUrl={services.application.getUrlForApp('management', {
+          path: 'stack/license_management',
+        })}
+      />
```

Comment:
Consider creating a small component inside `security_solution_ess/public/upselling/sections/` to wrap the component from the package, so we keep the registration clean:
```
component: EntityAnalyticsPanelUpsellingSectionLazy
```